frontend/src/component/layout/MainLayout/NavigationSidebar/NewInUnleash/NewInUnleashSideBarItem.tsx

```diff
@@ -0,0 +1,90 @@
+import type * as React from 'react';
```

Comment:
This component isn't really new code. It's stealing the responsibility for rendering a sidebar item from the NewInUnleashItem